integration-sandbox/packages/contracts/test/index.test.ts

```diff
@@ -71,8 +71,8 @@ describe("arrays", async () => {
     await expect(pushManyButton).toBeVisible();
     await expect(pushOneButton).toBeVisible();
 
-    // make sure scripts are loaded
-    await page.waitForLoadState("load");
+    // make sure setup is finished before clicking buttons
+    await expect(page.getByTitle("Setup status")).toHaveText("finished");
```

Comment:
Where does the title "Setup status" come from? Should we use the element id (`setup-status`) instead of the title here?

Reply:
btw I couldn't strictly use `id` itself, that's why some elements there have `data-testid`, which playwright uses instead